stories/Collapsible.stories.js

```diff
@@ -61,3 +61,52 @@ stories.add('Expandable', () => (
     </CollapsibleItem>
   </Collapsible>
 ));
+
+stories.add('Popout', () => (
+  <Collapsible popout>
+    <CollapsibleItem
+      header="Better safe than sorry. That's my motto."
+      icon="filter_drama"
+    >
+      Better safe than sorry. That's my motto.
+    </CollapsibleItem>
+    <CollapsibleItem
+      header="Yeah, you do seem to have a little 'shit creek' action going."
+      icon="place"
+    >
+      Yeah, you do seem to have a little 'shit creek' action going.
+    </CollapsibleItem>
+    <CollapsibleItem
+      header="You know, FYI, you can buy a paddle. Did you not plan for this contingency?"
+      icon="whatshot"
+    >
+      You know, FYI, you can buy a paddle. Did you not plan for this
+      contingency?
+    </CollapsibleItem>
+  </Collapsible>
+));
+
+stories.add('Preselected Section', () => (
+  <Collapsible>
+    <CollapsibleItem
+      header="Better safe than sorry. That's my motto."
+      icon="filter_drama"
+    >
+      Better safe than sorry. That's my motto.
+    </CollapsibleItem>
+    <CollapsibleItem
+      header="Yeah, you do seem to have a little 'shit creek' action going."
+      icon="place"
+      className="active"
```

Comment:
should this be preselected? or how should we tackle this?

Reply:
@alextrastero I'm sorry, I don't grasp what are you trying to say :smile:

Reply:
hehe sorry, shouldn't one of these components have an interface to **be** preselected?

Reply:
Sorry but I still don't understand :smile:

Reply:
`CollapsibleItem` has a prop (interface) `expanded` which has the same effect of adding the class `active`, let's use that prop best :)

Reply:
Yep, you are right @alextrastero , sorry for the misunderstanding